### PR TITLE
2.15 - clean ups

### PR DESCRIPTION
**Pull request by SpacePossum**
Just some stuff I found over the last weeks when trying to wrap up for the new releases.

### CODE REVIEW

File: src/Console/Command/DescribeCommand.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $name = $input->getArgument('name');
+        $name = rtrim($input->getArgument('name'), " \t,");
```
Comment (julienfalque) [inline src/Console/Command/DescribeCommand.php]: What is the purpose of this change?

File: src/Fixer/ClassNotation/OrderedClassElementsFixer.php [hunk 2]
```diff
@@ -232,7 +232,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $i = $tokens->getNextTokenOfKind($i, ['{']);
             $elements = $this->getElements($tokens, $i);
 
-            if (!$elements) {
+            if (0 === \count($elements)) {
```
Comment (julienfalque) [inline src/Fixer/ClassNotation/OrderedClassElementsFixer.php]:
```suggestion
            if ([] === $elements) {
```

File: tests/AutoReview/TravisTest.php [hunk 3]
```diff
@@ -113,11 +113,9 @@ private function getAllPhpVersionsUsedByCiForDeployments()
             return 'Deployment' === $job['stage'];
         });
 
-        $versions = array_map(function ($job) {
+        return array_map(function ($job) {
```
Comment (julienfalque) [inline tests/AutoReview/TravisTest.php]: Shouldn't this be handled by `return_assignment`?

File: src/Console/Command/DescribeCommand.php (hunk 1 above)
Comment (SpacePossum) [reply src/Console/Command/DescribeCommand.php]: basically to support this;
```
$ php-cs-fixer describe braces,

In DescribeCommand.php line 119:
                                                    
  Rule "braces," not found. Did you mean "braces"?  
                                                    

describe <name>
```

which happens a lot when you are copy-pasting from a list

Comment (julienfalque) [reply src/Console/Command/DescribeCommand.php]: Meh, it still feels weird to silently fix user input this way, I would keep the current behavior.

Comment (SpacePossum) [reply src/Console/Command/DescribeCommand.php]: why, the trailing comma and trailing space can never be part of the fixer name by design, so why not ignore?

File: src/Fixer/ClassNotation/OrderedClassElementsFixer.php (hunk 2 above)
Comment (SpacePossum) [reply src/Fixer/ClassNotation/OrderedClassElementsFixer.php]: -1, I prefer to not create an array when I don't need it.

File: tests/AutoReview/TravisTest.php (hunk 3 above)
Comment (SpacePossum) [reply tests/AutoReview/TravisTest.php]: could be, maybe best to open a new ticket on it though

File: src/Console/Command/DescribeCommand.php (hunk 1 above)
Comment (julienfalque) [reply src/Console/Command/DescribeCommand.php]: Because of [Fail-fast principle](https://en.wikipedia.org/wiki/Fail-fast): when there is an error, we should report it immediatly instead of trying to hide it silently.

If we want to go this way, maybe there are a lot of other places in the code where we could add such workarounds. Also, this change requires a test. All of this means more maintenance work, not worth it at all IMO.

Comment (SpacePossum) [reply src/Console/Command/DescribeCommand.php]: I don't consider a trailing comma an error, like PHP supports it in array lists as well. I think here UX is more important than a theory

Comment (julienfalque) [reply src/Console/Command/DescribeCommand.php]: Well, I do prefer when a tool tells me when I make a mistake instead of hiding it and letting me continue with my wrong usage.

Comment (SpacePossum) [reply src/Console/Command/DescribeCommand.php]: sure, I just want to support `name,` because I think it is valid usage
